package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "atom",
+  "author": "Atom Community <atom@atom.com.br>",
```

Comment:
The correct email is [atomcommunity@outlook.com](mailto:atomcommunity@outlook.com)